### DISCUSSION TITLE
how we can get raw lldp pdu data

### DISCUSSION THREAD

**Discussion by cbalhara** [lldpd/lldpd#574] [General]
Hi LLDP experts,

I just wanna to know how we can get RAW (byte stream)LLDP PUD data using lldpcli.
I tried `lldpcli -f json0/plain/keyvalue` with these options but its not helpful.

Could you please suggest and other configuration by which i can get byte stream.

thanks

**Comment by vincentbernat** [comment]
That's not possible.

**Reply by cbalhara** [reply]
Thanks [vincentbernat](https://github.com/vincentbernat) for the reply.

Is there any other way/method by which we can get TLV type and subtype as hex value

**Reply by vincentbernat** [reply]
Unknown TLVs are exposed as is by lldpcli. For existing ones, no. You could just use tcpdump?

**Reply by cbalhara** [reply]
thanks.
